tux/utils/constants.py

```diff
@@ -1,40 +1,34 @@
+import tomllib
+from pathlib import Path
 from typing import Final
 
-# TODO: move to assets/data/ potentially
+# Get the absolute path to the assets/data directory
+_ASSETS_DATA_PATH = Path(__file__).parent.parent.parent / "assets" / "data"
```

Comment:
theres a variable that makes getting data path easier from config.py which is more bulletproof incase of file relocation